src/app/launcher/step-indicator/step-indicator.component.ts

```diff
@@ -82,6 +82,9 @@ export class StepIndicatorComponent implements OnInit {
     this.launcherComponent.validateProjectName();
   }
 
+  @broadcast('stepIndicatorProjectInputClicked', {})
```

Comment:
shouldn't it be `stepIndicatorClicked`?

Reply:
No what I have here is different from `stepIndicatorClicked`. This event is to understand if the users are really using the project name input box in the left navigation